package.json

```diff
@@ -49,6 +49,7 @@
     "file-loader": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^23.6.0",
+    "jquery": "^3.3.1",
```

Comment:
Why do we need jquery?

Reply:
@Kachulio1 We don't need it I will remove it.